requirements_all.txt

```diff
@@ -4,4 +4,5 @@ ed25519
 zeroconf
 curve25519-donna
 pyqrcode
-base36
\ No newline at end of file
+base36
+aiohttp==3.1.1
```

Comment:
I don't like that we add a dependency that is only used for an optional accessory. If that's the way to go, we should move all optional accessories elsewhere #43.